utilities/tools/Get-RoleAssignmentList.ps1

```diff
@@ -57,19 +57,24 @@ function Get-RoleAssignmentList {
         if ("$ProviderNamespace/$ResourceType" -eq 'Microsoft.Authorization/RoleAssignments') {
             # No filter
             $relevantRoles = $roleDefinitions
-        } else {
+        }
+        else {
             # Filter Action based
             $relevantRoles += $roleDefinitions | Where-Object {
                 $_.Actions -like "$ProviderNamespace/$ResourceType/*" -or
                 $_.Actions -like "$ProviderNamespace/`**" -or
-                $_.Actions -like '`**'
```

Comment:
Hey @segraef, much appreciated. The change looks good to me - but - if we apply it it would be good to update the documentation a bit as well as it currently claims to return 'all' available role assignments which is simply not true. I guess it changes more towards 'relevant' or similar.

Reply:
I'd suggest to keep the functionality to return all possible roles and instead introduce a switch to only return the relevant ones. We can turn the switch on by default.

Apart from that, great improvement 👏

Reply:
Incorporated both your requests, please review.

Reply:
Looks good to me. Just 2 suggestions for the description